### PR TITLE
Fix sccs deprecated division outside calc

### LINKED ISSUE #237: [Webpack Warning] -  deprecated division scss method
### Overview
The prestashop project imports the `_variables.scss` file from prestashop-ui-kit into several theme files and warnings appear at the time of the webpack build regarding the deprecated scss division method

### Solution
Used the division in the `calc() `method

### PR DESCRIPTION
**Pull request by mattgoud**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | The prestashop project imports the _variables.scss file from prestashop-ui-kit into several theme files and warnings appear at the time of the webpack build regarding the deprecated scss division method
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #237
| Sponsor company   | Your company or customer's name goes here (if applicable).
| How to test?      | Indicate how to verify that this change works as expected.

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hello @mattgoud!

This is your first pull request on [prestashop-ui-kit](https://github.com/PrestaShop/prestashop-ui-kit/) repository of the PrestaShop project.

Thank you, and welcome to this Open Source community!

**Review by tblivet — DISMISSED** [review]
I think you can remove the parentheses around `calc`, as they are not necessary.

**Review by tblivet — APPROVED** [review]
👌

**Comment by mattgoud** [comment]
> 👌

thanks @tblivet

**Comment by florine2623** [comment]
QA functional not needed here :)

**Review by jolelievre — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
